packages/cli/src/commands/generate/page/page.js

```diff
@@ -123,6 +123,12 @@ export const builder = (yargs) => {
       type: 'boolean',
       default: true,
     })
+    .option('typescript', {
```

Comment:
@dac09 is this the only generator that's not using `import { yargsDefaults } from '../../generate'` to set the option for typescript? Seems like it at first glance. Suggesting we do that here if possible to standardize.

Reply:
Yeah that's what caught me out! Let me see if its a quick fix.